test/10_isless.jl

```diff
@@ -60,6 +60,28 @@ v3 = CategoricalArrays.catvalue(3, pool)
     @test isless(v3, v2) === false
     @test isless(v3, v3) === false
 
+    @testset "comparison with a value" begin
+        @test isless(v1, 1) === false
+        @test isless(v1, 2) === true
+        @test_throws KeyError isless(v1, 10)
+        @test_throws KeyError isless(v1, "a")
+        @test isless(1, v1) == false
```

Comment:
fixed. Is there any benefit except for consistency across the file?

Reply:
AFAIK doing `== false` is actually less strict than not doing any equality check, since `@test` requires a `Bool` but `==` could be overloaded. Using `=== false` should be strictly equivalent to not doing any check. In practice only consistency matters.

Reply:
Good point 👍. Thanks